updates.jenkins.io.tf

```
@@ -17,6 +17,12 @@ resource "azurerm_storage_account" "updates_jenkins_io" {
   tags = local.default_tags
 }
 
+resource "azurerm_storage_share" "updates_jenkins_io" {
+  name                 = "updates-jenkins-io"
+  storage_account_name = azurerm_storage_account.updates_jenkins_io.name
+  quota                = 2 # Current updates.jenkins.io total size in /www/updates.jenkins.io: 400Mo
```

```suggestion
  quota                = 2 # Expected updates.jenkins.io total size in /www/updates.jenkins.io: 400Mo
```

Current deos not map to a specific timestamp ;)

---

WDYT of
```suggestion
  quota                = 2 # updates.jenkins.io total size in /www/updates.jenkins.io: 400Mo (Mid 2023)
```